dead_mans_switch_server/security/res_groups.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="group_user" model="res.groups">
+            <field name="name">User</field>
```

Comment:
note the group's category_id. The user will see Monitoring / User, that's in line with what core modules do too: https://github.com/OCA/OCB/blob/8.0/addons/project/security/project_security.xml#L5